graylog2-web-interface/src/views/components/visualizations/ChartData.js

```diff
@@ -53,7 +53,7 @@ export const extractSeries = (keyJoiner: KeyJoiner = _defaultKeyJoiner, leafValu
     flatLeafs.forEach(([key, value]) => {
       const joinedKey = keyJoiner(value.key);
       const targetIdx = xLabels.findIndex(l => isEqual(l, key));
-      if (value.value) {
+      if (value.value === 0 || value.value) {
```

Comment:
What about testing that it's neither `undefined` nor `null` at this point? Otherwise we would need to check for other falsy values as well.

```suggestion
      if (value.value !== null && value.value !== undefined) {
```

Reply:
What would be the benefit of including falsy values like an empty string or NaN?

Reply:
`extractSeries` is meant to be reused, so we should take care of its single purpose of extracting values. `null` or `undefined` would mean the absence of a value, while `''` `NaN` or `false` do not.

Reply:
Alright!